.github/workflows/ci.yml

```diff
@@ -87,3 +87,33 @@ jobs:
       - run: cargo run -- --help
       - run: cargo run -- list-remote node
       - run: cargo run -- list-remote wasm-test
+  build:
+    name: Upload build
+    runs-on: ${{ matrix.host }}
+    strategy:
+      matrix:
+        include:
+        - target: aarch64-unknown-linux-gnu
```

Comment:
this is wrong? `cargo install` is probably installing the host target, which is x86_64